lib/caching-precompiler.js

```diff
@@ -159,14 +160,12 @@ CachingPrecompiler.prototype._createEspowerPlugin = function () {
 };
 
 CachingPrecompiler.prototype._createTransform = function () {
-	var dependencies = {
-		'babel-plugin-espower': require('babel-plugin-espower/package.json').version,
-		'ava': require('../package.json').version,
-		'babel-core': require('babel-core/package.json').version,
-		'babelConfig': this.babelConfig
-	};
-
-	var salt = new Buffer(JSON.stringify(dependencies));
+	var salt = md5Hex([
+		packageHash.sync(require.resolve('../package.json')),
+		packageHash.sync(require.resolve('babel-core/package.json')),
+		packageHash.sync(require.resolve('babel-plugin-espower/package.json')),
+		JSON.stringify(this.babelConfig)
```

Comment:
Allowing arrays with holes in them, for this type of situation. We could just not call `.update()` when we find a hole. Or call update with a special placeholder set of data (so that the presence and position of holes in the array affects the final hash).

Still, probably best just to fix it here

Not sure it's a good idea.